### PR TITLE
feat: support llmspec format

### PR DESCRIPTION
**Pull request by cutecutecat**
- use `modelz-py` as client to make a guild.
- use `llmspec` format for `Request` and `Response`.
- export `NormalizeVideo` from `pytorch` to avoid deprecate warning.

ref: https://github.com/pytorch/vision/blob/0ab7d05c5202e8620000566b6ea7925eae9146e1/torchvision/transforms/_transforms_video.py#L22

Fixes MZ-140

### CODE REVIEW

File: example/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+modelz
```
Comment (kemingy) [inline example/requirements.txt]: This is not the TensorChord one
```suggestion
modelz-py
```

File: data.py [hunk 2]
```diff
@@ -5,27 +5,50 @@
...
+    def __init__(self, mean, std):
+        self.mean = mean
+        self.std = std
+
+    def __call__(self, clip):
+        """
+        Args:
+            clip (torch.tensor): video clip to be normalized. Size is (C, T, H, W)
+        """
+        clip = clip.clone()
```
Comment (kemingy) [inline data.py]: It's better to rename it.

File: data.py [hunk 3]
```diff
@@ -5,27 +5,50 @@
...
-from torchvision.transforms._transforms_video import NormalizeVideo
+
+from models.multimodal_preprocessors import SimpleTokenizer
 
 DEFAULT_AUDIO_FRAME_SHIFT_MS = 10  # in milliseconds
 
 BPE_PATH = "bpe/bpe_simple_vocab_16e6.txt.gz"
 
 
+class NormalizeVideo:
```
Comment (kemingy) [inline data.py]: Is it similar to https://torchvideo.readthedocs.io/en/latest/transforms.html#torchvideo.transforms.NormalizeVideo?

File: example/requirements.txt (hunk 1 above)
Comment (cutecutecat) [reply example/requirements.txt]: Fixed.

File: data.py (hunk 3 above)
Comment (cutecutecat) [reply data.py]: yes, but it's a bit wasted to introduce a whole package for one function, I found we could use `pytorchvideo.transforms.Normalize`, which is already imported. I have tested their output is equal.

File: data.py (hunk 2 above)
Comment (cutecutecat) [reply data.py]: Deprecated.

**Review by gaocegege — APPROVED** [review]
